internal/tracing/manifests/values.go

```diff
@@ -0,0 +1,33 @@
+package manifests
+
+import (
+	"encoding/json"
+
+	"k8s.io/klog/v2"
+)
+
+type TracingValues struct {
+	Enabled     bool   `json:"enabled"`
+	OTELColSpec string `json:"otelColSpec"`
+}
+
+func BuildValues(opts Options) (TracingValues, error) {
+	values := TracingValues{
+		Enabled: true,
+	}
+
+	klog.Info("Building OTEL Collector instance")
+	otelColSpec, err := buildOtelColSpec(opts)
+	if err != nil {
+		return values, err
+	}
+
+	b, err := json.Marshal(otelColSpec)
+	if err != nil {
+		return values, err
```

Comment:
shouldn't this return an empty object? I would change the API to return a pointer